interfaces/apparmor/spec.go

```diff
@@ -31,6 +31,16 @@ import (
 	"github.com/snapcore/snapd/strutil"
 )
 
+// unconfined mode is only enabled when the interface supports it as a static
+// property and it is then enabled via EnableUnconfined() method
```

Comment:
s/EnableUnconfined/SetUnconfinedEnabled/ ?

Reply:
Also fixed as part of 0b7a92c01e